nginx/user_conf.d/default.conf

```diff
@@ -19,21 +43,87 @@ server {
     ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
 
     # SSL sessions
-    ssl_session_timeout 1d;
+    ssl_session_timeout 30m;
     # ssl_session_cache dfine in stream and http
     ssl_session_tickets off;
+    ssl_session_cache shared:SSL:30m;
+    # this is the internal Docker DNS, cache only for 30s
+    resolver 127.0.0.11 valid=30s;
+    ssl_stapling on;
+    ssl_stapling_verify on;
+    ssl_buffer_size 8k;
+
+    # Let nginx start if upstream host is unavailable or down
+    set $ckan ckan:5000;
```

Comment:
Allow Nginx to start even if the upstream (ckan) service is unavailable.
Without this, Nginx may fail to start/restart when the upstream host is down.
Enabling dynamic resolving ensures Nginx can still start independently.